ui/app/mainui/activitycenter/views/ActivityNotificationBase.qml

```diff
@@ -21,10 +21,6 @@ Item {
 
     StatusFlatRoundButton {
         id: markReadBtn
-        width: 32
-        height: 32
-        icon.width: 24
-        icon.height: 24
         anchors.verticalCenter: parent.verticalCenter
```

Comment:
there's no `parent` around

Reply:
Root item is parent for StatusFlatRoundButton